src/data/metadata.csv

```diff
@@ -9053,6 +9052,741 @@ IN_TR,IN,India,TR,Tripura,,,,,,
 IN_UP,IN,India,UP,Uttar Pradesh,,,,,,
 IN_UT,IN,India,UT,Uttarakhand,,,,,,
 IN_WB,IN,India,WB,West Bengal,,,,,,
+IN_AN_NMA,IN,India,AN,Andaman and Nicobar Islands,NMA,North and Middle Andaman,,,North and Middle Andaman,
+IN_AN_NIC,IN,India,AN,Andaman and Nicobar Islands,NIC,Nicobars,,,Nicobars,
```

Comment:
No need for a `match_string` if it's the same as `subregion2_name`

Reply:
Done.